tests/modelClasses.py

```diff
@@ -39,6 +60,15 @@ class Customer(User):
     is_admin: Optional[bool]
     lists: Optional[list[int]]
 
+    @staticmethod
+    def parse_name(name: str) -> str:
```

Comment:
Hi @andrea-mucci, Another topic I want to comment with you is that for technical simplicity I imposed that parsing methods must be static (or class) methods. 

The reason why I did that is to ensure that only a single parameter is passed (and ease the internal logic),  but I want to ask you if this approach is right according to the future use cases of this feature. I mean, with the current strategy we couldn't get any other attribute of the current class instance.

Reply:
For the parser i suppose that only one parameter must be parsed, this is not the same for validation process.

During the daily we have to expose some use cases and check if is the right way for the parser to get only one parameter

Reply:
Okay @andrea-mucci! I'll proceed to apply the changes as we discussed during the daily meeting. 🙂